devito/symbolics.py

```diff
@@ -7,15 +7,25 @@
 All exposed functions are prefixed with 'dse' (devito symbolic engine)
 """
 
+from __future__ import absolute_import
+
+from collections import OrderedDict
+
 import numpy as np
-from sympy import (Add, Eq, Function, Indexed, IndexedBase, Symbol, cse,
-                   lambdify, numbered_symbols, preorder_traversal, symbols)
+import sympy
```

Comment:
nope, because they would conflict with ``class Mul`` and ``class Add`` at the bottom of this module (I'm extending the sympy hierarchy, as explained in the comments below)